repotracker/repotracker.go

```diff
@@ -589,3 +595,67 @@ func createVersionItems(v *version.Version, ref *model.ProjectRef, project *mode
 	}
 	return nil
 }
+
+func addBuildBreakSubscriptions(v *version.Version, projectRef *model.ProjectRef) error {
+	subscriptionBase := event.Subscription{
+		Type:      event.ResourceTypeVersion,
+		Trigger:   "regression",
+		Selectors: trigger.MakeVersionSelectors(*v),
+	}
+	subscribers := []event.Subscriber{}
+
+	// if the commit author wants build break notifications, subscribe them to this commit
+	if v.AuthorID != "" {
+		subscriber, err := makeBuildBreakSubscriber(v.AuthorID)
+		if err != nil {
+			return err
+		}
+		if subscriber != nil {
+			subscribers = append(subscribers, *subscriber)
+		}
+	}
+	// if the project has build break notifications, subscribe admins if no one subscribed
+	if projectRef.NotifyOnBuildFailure && len(subscribers) == 0 {
+		for _, admin := range projectRef.Admins {
+			subscriber, err := makeBuildBreakSubscriber(admin)
+			if err != nil {
+				return err
```

Comment:
Catch and try to make subscriptions for the other project admins.

Reply:
done